packages/Table/src/TableHead/SortableCell/SortableCell.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+
+const SortableCell = ({
+  onSort,
+  orderBy,
+  column,
+  columns
+}: {
+  onSort: any;
+  orderBy: any;
+  column: any;
+  columns: any;
+}) => {
+  const formatColumns = columns.reduce((acc: any, item: any): any => {
+    return { ...acc, [item]: null };
+  }, {});
+  const [tableColumns, setTableColumns] = useState(formatColumns);
+
+  type TSortDirection = "asc" | "desc" | null;
+  const asc: TSortDirection = "asc";
```

Comment:
We could move this consts to a constants file if you think we are going to reuse these values in the future